packages/nix/NIX.md

```diff
@@ -0,0 +1,26 @@
+# How to import
+
+To import with flakes use
+```nix
+inputs = {
+  polymc.url = "github:PolyMC/PolyMC";
+};
+
+...
+
+nixpkgs.overlays = [ inputs.polymc.overlay ]; ## Within configuration.nix
```

Comment:
#31 uses flake-utils so overlay attrs are `inputs.polymc.overlay.x86_64-linux` and `inputs.polymc.overlay.aarch64-linux`
PR only provides flake overlays. `polymc` also will be packaged for nixpkgs so non-flake users can get it from hydra.

Reply:
Changed the inputs section to `inputs.poiymc.overlay.${system}` which should reflect it more accurately